src/functions.js

```diff
@@ -88,6 +88,42 @@ defineFunction(["\\color"], {
     argTypes: ["color"],
 }, null);
 
+// colorbox
+defineFunction(["\\colorbox"], {
+    numArgs: 2,
+    allowedInText: true,
+    greediness: 3,
+    argTypes: ["color", "text"],
+}, function(context, args) {
+    const color = args[0];
+    const body = args[1];
+    return {
+        type: "enclose",
+        label: context.funcName,
+        bkgrndColor: color,
```

Comment:
I have a slight preference for `backgroundColor` mainly b/c I type it a lot so it easier to type.